frontend/app/services/lookup-service.server.ts

```diff
@@ -9,6 +9,19 @@ const preferredLanguageSchema = z.object({
   nameFr: z.string().optional(),
 });
 
+const countrySchema = z.object({
+  code: z.string(),
+  nameEn: z.string().optional(),
+  nameFr: z.string().optional(),
+});
+
+const regionSchema = z.object({
+  code: z.string(),
+  countryCode: z.string(),
+  nameEn: z.string().optional(),
+  nameFr: z.string().optional(),
```

Comment:
I don't see why `nameEn` and `nameFr` should be optional. The APIs we are dealing with should be able to provide countries and regions, in both official languages.

Reply:
fixed